spec/unit/resource/api1000/synergy/hypervisor_cluster_profile_spec.rb

```diff
@@ -0,0 +1,20 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1000::Synergy::HypervisorClusterProfile do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API800::Synergy::HypervisorClusterProfile' do
+    expect(described_class).to be < OneviewSDK::API800::Synergy::HypervisorClusterProfile
```

Comment:
I think it should inherit API1000::C7000 class. If so, please update description also accordingly.

Reply:
The respective lib file for API1000::Synergy file inherits from API800::Synergy file, so mentioned the same in spec file.

Reply:
okay